components/locale-chooser.js

```diff
@@ -57,3 +130,40 @@ function getLocaleAndHost() {
     return {locale: fallbackLocale, host}
   }
 }
+
+function mapLocale(key) {
+  const svgStyle = {width: '1em', height: '100%'}
+  switch (key) {
+    case 'en':
+      return {
+        key,
+        display: 'English',
+        flag: <EnSvg {...svgStyle} />,
+      }
+    case 'es':
+      return {
+        key,
+        display: 'Español',
+        flag: <EsSvg {...svgStyle} />,
+      }
+    case 'fr': {
+      return {
+        key,
+        display: 'Français',
+        flag: <FrSvg {...svgStyle} />,
```

Comment:
Yeah, the dynamic require was probably not something that'd work here (because we're actually using a babel plugin for this) 😄